content_schemas/formats/organisation.jsonnet

```diff
@@ -140,7 +140,20 @@
                       type: "string",
                     },
                     image: {
-                      "$ref": "#/definitions/image",
+                      "oneOf": [
+                        {
+                          "$ref": "#/definitions/image",
+                        },
+                        {
+                          "type": "null"
+                        }
```

Comment:
I wonder if we need this to be allowed to be null. As this is not a required property we just need to not set the attribute rather than sending a null value.